app/config/axios.js

```diff
@@ -1,5 +1,5 @@
 import axios from "axios";
 
 export default axios.create({
-  baseURL: 'https://plus1up.herokuapp.com/api',
+  baseURL: 'https://plus1up-staging-pr-10.herokuapp.com/api/',
```

Comment:
I've successfully promoted changes to staging, please change to `https://plus1up-staging.herokuapp.com/` and confirm if everything works fine